ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/LongRangeAttack.hs

```diff
@@ -14,50 +14,40 @@ import           Ouroboros.Consensus.Block.Abstract (HeaderHash)
 import           Ouroboros.Consensus.Util.Condense (condense)
 import           Ouroboros.Network.AnchoredFragment (headAnchor)
 import qualified Ouroboros.Network.AnchoredFragment as AF
-import           System.Random.Stateful (runSTGen_)
 import           Test.Consensus.Genesis.Setup
 import           Test.Consensus.Genesis.Setup.Classifiers
 import           Test.Consensus.PeerSimulator.Run (noTimeoutsSchedulerConfig)
 import           Test.Consensus.PeerSimulator.StateView
 import           Test.Consensus.PointSchedule
 import qualified Test.QuickCheck as QC
 import           Test.QuickCheck
-import           Test.QuickCheck.Extras (unsafeMapSuchThatJust)
-import           Test.QuickCheck.Random (QCGen)
 import           Test.Tasty
 import           Test.Tasty.QuickCheck
 import           Test.Util.Orphans.IOLike ()
 import           Test.Util.TestBlock (TestBlock, unTestHash)
+import           Test.Util.TestEnv (adjustQuickCheckTests)
 
 tests :: TestTree
 tests =
   testGroup "long range attack" [
-    testProperty "one adversary" (prop_longRangeAttack 1 [10])
+    adjustQuickCheckTests (`div` 10) $
+    testProperty "one adversary" prop_longRangeAttack
     -- TODO we don't have useful classification logic for multiple adversaries yet – if a selectable
     -- adversary is slow, it might be discarded before it reaches critical length because the faster
     -- ones have served k blocks off the honest chain if their fork anchor is further down the line.
     -- ,
     -- testProperty "three adversaries" (prop_longRangeAttack 1 [2, 5, 10])
   ]
 
-genChainsAndSchedule :: PointScheduleConfig -> Word -> ScheduleType -> QC.Gen (GenesisTest, PointSchedule)
-genChainsAndSchedule scheduleConfig numAdversaries scheduleType =
-  unsafeMapSuchThatJust do
-    gt <- genChains numAdversaries
-    seed :: QCGen <- arbitrary
-    pure ((gt,) <$> runSTGen_ seed (\ g -> genSchedule g scheduleConfig scheduleType (gtBlockTree gt)))
-
-newLRA :: Bool
-newLRA = True
-
-prop_longRangeAttack :: Int -> [Int] -> QC.Gen QC.Property
-prop_longRangeAttack honestFreq advFreqs = do
-  (genesisTest, schedule) <- genChainsAndSchedule scheduleConfig (fromIntegral (length advFreqs)) sched
+prop_longRangeAttack :: QC.Gen QC.Property
+prop_longRangeAttack = do
+  genesisTest <- genChains (pure 1)
```

Comment:
Perhaps we should explain why `1` was chosen here.

Reply:
Done in d96aee77387eccb196522b4e5b3df1b61518440c.